docs/v1/integrations/rest.mdx

```diff
@@ -0,0 +1,449 @@
+
+### Initial Authentication
+
+When you create a session, you'll use your API key and receive a JWT token in response:
+
+<CodeGroup>
+```bash curl
+curl -X POST https://api.agentops.ai/create_session \
+  -H "Content-Type: application/json" \
+  -H "X-Agentops-Api-Key: your_api_key" \
+  -d '{
+    "id": "550e8400-e29b-41d4-a716-446655440000",
+    "init_timestamp": "2024-03-14T12:00:00Z"
+  }'
+```
+
+```json Success Response
+{
+  "status": "success",
+  "jwt": "eyJhbGciOiJIUzI1NiIs...",
+  "session_id": "550e8400-e29b-41d4-a716-446655440000"
+}
+```
+
+```bash Error Response
+HTTP/1.1 401 Unauthorized
+{
+  "error": "Invalid API key",
+  "message": "Please check your API key and try again"
+}
+```
+</CodeGroup>
+
+### Using JWT Tokens
+
+Use the JWT token in the Authorization header for all subsequent requests:
+
+<CodeGroup>
+```bash Example Request
+curl -X POST https://api.agentops.ai/create_events \
+  -H "Authorization: Bearer eyJhbGciOiJIUzI1NiIs..." \
+  -H "Content-Type: application/json" \
+  -d '{
+    "events": [{
+      "type": "llm",
+      "init_timestamp": "2024-03-14T12:01:00Z"
+    }]
+  }'
+```
+
+```bash Error Response
+HTTP/1.1 401 Unauthorized
+{
+  "error": "Invalid or expired JWT",
+  "message": "Please reauthorize using /reauthorize_jwt"
+}
+```
+</CodeGroup>
+
+### Refreshing Tokens
+
+JWTs expire after 24 hours. When a token expires, use your API key to get a new one:
+
+<CodeGroup>
+```bash curl
+curl -X POST https://api.agentops.ai/reauthorize_jwt \
+  -H "Content-Type: application/json" \
+  -H "X-Agentops-Api-Key: your_api_key" \
+  -d '{
+    "session_id": "550e8400-e29b-41d4-a716-446655440000"
+  }'
+```
+
+```json Success Response
+{
+  "status": "success",
+  "jwt": "eyJhbGciOiJIUzI1NiIs..."
+}
+```
+</CodeGroup>
+
+## Session Management
+
+Sessions require a unique identifier that you generate client-side. While any unique string will work, we recommend using UUIDs for consistency. Here's how to generate one in Python:
+
+<CodeGroup>
+```python Python
+import uuid
+session_id = str(uuid.uuid4())
+# "550e8400-e29b-41d4-a716-446655440000"
+```
+</CodeGroup>
+
+### Create Session
+
+Start a new monitoring session using your generated session ID:
+
+<CodeGroup>
+```bash curl
+curl -X POST https://api.agentops.ai/create_session \
+  -H "Content-Type: application/json" \
+  -H "X-Agentops-Api-Key: your_api_key" \
+  -d '{
+    "id": "550e8400-e29b-41d4-a716-446655440000",
+    "init_timestamp": "2024-03-14T12:00:00Z",
+    "tags": ["production", "customer-service"],
+    "host_env": {
+      "python_version": "3.9.0",
+      "platform": "linux"
+    }
+  }'
+```
+
+```bash Request
+POST https://api.agentops.ai/create_session
+Content-Type: application/json
+X-Agentops-Api-Key: your_api_key
+
+{
+  "id": "550e8400-e29b-41d4-a716-446655440000",
+  "init_timestamp": "2024-03-14T12:00:00Z",
+  "tags": ["production", "customer-service"],
+  "host_env": {
+    "python_version": "3.9.0",
+    "platform": "linux"
+  }
```

Comment:
Check the how the host env is actually structured

Might be good to mention we only read specific values